packages/extension/src/ExtensionActions.ts

```diff
@@ -9,6 +9,7 @@ interface IMmiActionsFactoryParameters {
   callBackgroundMethod: any;
 }
 
+// TODO - This is not being used anywhere and it's duplicated in the extension
```

Comment:
This function is not being used. The same spec exists within the extension code. Should be deleted.